### PR TITLE
Order milestone updated

### PR DESCRIPTION
**Pull request by chiestroud**
## Description
- [x] User can select menu to specific reservation (fixed bug so the selected menu shows up every time)
- [x] All the ingredients from the selected menu will be reduced by 1
- [x] When ingredient's inventory becomes 0, every menu uses the ingredient becomes unavailable
- [x] Menu becomes available again when ingredients are restocked from ingredients page.

## Related Issue
#104 #105 

## How Can This Be Tested?
git fetch
git checkout cs-order-count

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by hmparsons-sde — APPROVED** [review]
Sorry for the delay!

**Review by jrobinson0529 — COMMENTED** [review]
Can you take the console warn out on ```navEvents.js?879b:39```

File: src/javascripts/events/domEvents.js [hunk 1]
```diff
@@ -131,18 +136,6 @@ const domEventListeners = (e) => {
       Promise.all(deletedArray).then((response) => console.warn(response));
```
Comment (jrobinson0529) [inline src/javascripts/events/domEvents.js:136]: Remove ```.then((response) => console.warn(response));```

**Review by jrobinson0529 — APPROVED** [review]
💯

**Pull request merged**